static/app/views/performance/browser/resources/resourceSummaryPage/sampleImages.tsx

```diff
@@ -125,9 +125,26 @@ function SampleImagesChartPanelBody(props: {
   return (
     <ImageWrapper>
       {images.map(resource => {
+        const hasRawDomain = Boolean(resource[RAW_DOMAIN]);
+        const isRelativeUrl = resource[SPAN_DESCRIPTION].startsWith('/');
+        let src = resource[SPAN_DESCRIPTION];
+        if (isRelativeUrl && hasRawDomain) {
+          try {
+            const url = new URL(resource[SPAN_DESCRIPTION], resource[RAW_DOMAIN]);
```

Comment:
l: Can we extract this out to be a utility function?

Reply:
Hmm, what parts specifically, the whole try catch block? The try catch block + the `isRelativeUrl && hasRawDomain` logic?

Reply:
The try catch block + the isRelativeUrl && hasRawDomain logic.

Ideally with a func signature that looks like the following:

```ts
function getUrlFromSpanDescription(description: string | undefined, domain: string | undefined): string | undefined;
```

but it's also `l` we dont have to come back and add this too.

Reply:
That's a good point, I can definitely see this being used elsewhere in the future.